toolkit/imageconfigs/scripts/cleanup.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -eux
+
+# cleanup symlinks created by the toolkit that are not needed for base images
+if [ -L /srv ]; then
+  echo "Removing /srv symlink"
+  rm /srv
+else
+  echo "/srv symlink does not exist"
+fi
+
+# cleanup any logs that may have been created during the build
```

Comment:
With this comment, it seems the intention is to just "rm -rf /var/log" if it exists / is a symlink.

Reply:
Here I'm actually clearing the logs directory and not removing symlinks

Reply:
ah actually I just noticed I got the check wrong. Will fix.